solutions/coreclr/Makefile

```diff
@@ -33,7 +33,7 @@ ext2fs: appdir
 	$(MYST) fssig --roothash ext2fs > roothash
 
 run:
-	./test-runner.sh $(MYST_EXEC) ext2 $(TEST_FILE)
+	COMPlus_EnableDiagnostics=0 COMPlus_EnableAlternateStackCheck=1 ./test-runner.sh $(MYST_EXEC) ext2 $(TEST_FILE)
```

Comment:
I thought we block the environment variables from host by default, unless we explicitly allow it in config.json. Did we change the behavior recently?

Reply:
I tested with a simple program. It does look like passing host env variables is the default.

Reply:
I think that's a regression then. In the [document](https://github.com/deislabs/mystikos/blob/main/doc/sign-package.md) it says:

>  HostEnvironmentVariables | A list of environment variables that can be imported from the insecure host

Without an explicit entry in config.json that's measured, we shouldn't rely on any environment variable from the untrusted host, even though that's convenient. 

The remedy here is to create a config.json and put these two variables in there and run myst with `--app-config-path`

Reply:
Please create an issue for the regression.

Reply:
> The remedy here is to create a config.json and put these two variables in there and run myst with --app-config-path

Fixed.